app/javascript/components/widgets/weather/semi-circle.jsx

```diff
@@ -1,12 +1,20 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import cityImage from '@images/buildings.png';
+import pvdImage from '@images/buildings.png';
+import bldImage from '@images/boulder.svg';
 import { yValue, xValue } from '@lib/circle-math';
 import { timeDiffInMinutes, timeAndDateForTimezone } from '@lib/datetime';
 import { colors } from '@lib/theme';
 import styled, { keyframes } from 'styled-components';
 import MoonPhase from '@weather/moon-phase';
 
+const getCityImage = () => {
+  if (window.location.pathname.includes('Boulder')) return bldImage;
+  return pvdImage;
+};
+
+const cityImage = getCityImage();
```

Comment:
This is a problem. This evaluates what the cityImage is at load time. I think we need to evaluate this **every time the route changes**.

Reply:
This should be fixed now!